### DISCUSSION TITLE
Without "Node Sets"

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#22744] [Q&A Meshing] [answered]
All,
I am using a mesh file generated by CUBIT and the MOOSE run generates a strange result.
I removed "Node Sets" and then the result looks good.
Please let me know if MOOSE can run without "Node Sets" while "Side Sets" are kept.

Please help,
S. Thomas

**Comment by GiudGiud** [comment, marked as answer]
Hello

We should be able to load both node sets and side sets from a cubit generated mesh.
There must be something else wrong, like the boundary conditions are not set on the proper sideset. 

Please note that MOOSE automatically converts nodesets to sidesets at the end of the mesh generation process
see this page
https://mooseframework.inl.gov/source/mesh/MooseMesh.html#more_detail

Guillaume

**Reply by styyokuda** [reply]
Thank you very much Guillaume.
I would like to use this mesh file without Node Sets because it seems working.
Please let me know if it is OK.
Sincerely,
S. Thomas

**Reply by GiudGiud** [reply]
If the mesh file has the side sets you need then it should work. 

you will get an error if you try to apply a boundary condition or an interface kernel on a boundary that does not exist

you should also be able to see what side set and node set exist in paraview so there s really no way to get it wrong
